OpenRA.Mods.Common/Traits/Attack/AttackGarrisoned.cs

```diff
@@ -185,9 +184,6 @@ public override void DoAttack(Actor self, Target target)
 					muzzles.Add(muzzleFlash);
 					muzzleAnim.PlayThen(sequence, () => muzzles.Remove(muzzleFlash));
 				}
-
-				foreach (var npa in self.TraitsImplementing<INotifyAttack>())
-					npa.Attacking(self, target, a, barrel);
```

Comment:
Camo pillboxes no longer uncloak while attacking, so the implementation in `Armament` is somehow not covering this case!

Reply:
I guess that `Armament` will be notifying the passenger, while this is needed to notify the pillbox. Adding this back to just `AttackGarrisoned` should hopefully be sufficient.

Reply:
Fixed.